docs/includes/net-standard-table.md

```diff
@@ -2,7 +2,7 @@
 |-----------------------------------|-------|--------|-------|-------|-------|--------|--------|-------|
 | .NET Core                         | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0    | 1.0    | 2.0   |
 | .NET Framework (with tooling 1.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2  | vNext  | 4.6.1 |
-| .NET Framework (with tooling 2.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.61   | 4.6.1  | 4.6.1 |
+| .NET Framework (with tooling 2.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1   | 4.6.1  | 4.6.1 |
```

Comment:
Very tiny nit, but we're trying to maintain tables in clean format everywhere (and fix the bad ones as we go). Could you take out a space from the cell to get the column back in alignment?